### DISCUSSION TITLE
Detected: Trojan:Win32/Casdet!rfn

### DISCUSSION THREAD

**Discussion by bickle** [valinet/ExplorerPatcher#3450] [General]
Detected: Trojan:Win32/Casdet!rfn

Since it is always the ep_setup.exe program that gets flagged, can you not just get rid of it and have users do manual installs? It's crazy how much this gets flagged. Obviously this behavior is just going to get flagged again and again.

Build 22621.3527.65.5

Detected: Trojan:Win32/Casdet!rfn

file: C:\Program Files\ExplorerPatcher\ep_setup.exe
regkey: HKLM\SOFTWARE\MICROSOFT\WINDOWS\CURRENTVERSION\UNINSTALL\{D17F1E1A-5919-4427-8F89-A1A8503CA3EB}_ExplorerPatcher

uninstall: HKLM\SOFTWARE\MICROSOFT\WINDOWS\CURRENTVERSION\UNINSTALL\{D17F1E1A-5919-4427-8F89-A1A8503CA3EB}_ExplorerPatcher

**Comment by pyrates999** [comment]
All builds are done by github build servers using the source code in this repository.  You are free to check the source code yourself.

**Reply by bickle** [reply]
I assume it is a false positive. However, that doesn’t matter. As long as it gets flagged by MS, it generates the ire of IT and they come to me. 

The reoccurrence of flagging is the reason I’m asking about just ditching the setup executable (or at least the behavior that keeps gettin flagged). It seems like it is more trouble than it’s worth.

**Reply by pyrates999** [reply]
I wouldn't be installing this on work computers as you could lose your admin privileges at any time and then you can't uninstall it or update it.

**Comment by Amrsatrio** [comment]
If you are tired of false positives and have 5GB of spare space on your storage then our lord and savior Valinet has pieced together a powershell script to build EP right on your system even when you don't have Visual Studio stuff installed: https://github.com/valinet/ep_make

Though however I haven't added the lines to include EP's taskbar into the final exe, that is for later for me.
